tests/iio_attr.c

```diff
@@ -646,23 +663,27 @@ int main(int argc, char **argv)
 	if (search_context) {
 		unsigned int nb_ctx_attrs = iio_context_get_attrs_count(ctx);
 		if (!attr_index && nb_ctx_attrs > 0)
-			printf("IIO context with %u attributes:\n", nb_ctx_attrs);
+			printf("IIO context with %u attributes:\n",
+			       nb_ctx_attrs);
 
 		for (i = 0; i < nb_ctx_attrs; i++) {
 			const char *key, *value;
 			ssize_t ret;
 
 			ret = iio_context_get_attr(ctx, i, &key, &value);
 			if (!ret) {
-				if (!attr_index || str_match(key, argv[attr_index], ignore_case)) {
+				if (!attr_index ||
+				    str_match(key, argv[attr_index],
+					      ignore_case)) {
```

Comment:
this is another place where the 80-col-width limit seems to cause funky-ness;
maybe we should split it into another function